.github/workflows/release.yaml

```diff
@@ -48,6 +48,14 @@ jobs:
           COSIGN_PASSWORD: ${{ secrets.COSIGN_PASSWORD }}
           COSIGN_PRIVATE_KEY: ${{ secrets.COSIGN_PRIVATE_KEY }}
 
+      - name: Add Attestation Artifacts (binaries)
+        run: |
+          echo -n '${{ steps.release.outputs.artifacts }}' | jq -r '.[] | select(.type=="Binary" and .goos=="linux") | { "name": "\(.extra.ID)-\(.goos)-\(.goarch)", "path":"\(.path)"} | @base64' | while read i; do
```

Comment:
This code just extracts  the binaries path and calls `chainloop attestation add`

if you can think of a simpler way, I am all ears!

Reply:
Fine for now, I can play around with it.

Can you give me an example `{{ steps.release.outputs.artifacts }}` so I can experiment with it later this week?

Reply:
You can get it from [the job](https://github.com/goket-app/goket/actions/runs/3841373075/jobs/6541516633), `Artifacts output` header.

![image](https://user-images.githubusercontent.com/24523/210659861-ceaf6933-14f8-4358-a7b8-1cf5c946abf4.png)